robot/web/stream/camera2/camera.go

```diff
@@ -0,0 +1,120 @@
+// Package camera provides utilities for working with camera resources in the context of streaming.
+package camera
+
+import (
+	"context"
+	"fmt"
+	"image"
+
+	"github.com/pion/mediadevices/pkg/prop"
+
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/gostream"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/robot"
+	rutils "go.viam.com/rdk/utils"
+)
+
+var streamableImageMIMETypes = map[string]interface{}{
+	rutils.MimeTypeRawRGBA:  nil,
+	rutils.MimeTypeRawDepth: nil,
+	rutils.MimeTypeJPEG:     nil,
+	rutils.MimeTypePNG:      nil,
+	rutils.MimeTypeQOI:      nil,
+}
```

Comment:
I added an f.name print in [sniff](https://go.googlesource.com/go/+/go1.16/src/image/format.go#71)

```
f.name ppm
f.name qoi
f.name bmp
f.name tiff
f.name tiff
f.name gif
f.name jpeg
```

In addition to the first four supported formats, we also support QOI.

Reply:
who uses `QOI`?